src/main/java/org/alliancegenome/curation_api/auth/AuthenticationFilter.java

```diff
@@ -86,9 +86,12 @@ public class AuthenticationFilter implements ContainerRequestFilter {
 	@Override
 	public void filter(ContainerRequestContext requestContext) throws IOException {
 		if (jsonWebToken.getClaimNames() != null) {
-			Person person = personService.findPersonByOktaId(jsonWebToken.getClaim("uid"));
+			Person person = null;
 			if (person == null) {
```

Comment:
person will always be null here, so could just have `Person person = validateUserTokenById();`